packages/e2e-tests/specs/performance/front-end-twentytwentythree.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { basename, join } from 'path';
+import { writeFileSync } from 'fs';
+
+/**
+ * WordPress dependencies
+ */
+import { activateTheme, createURL, logout } from '@wordpress/e2e-test-utils';
+
+describe( 'Front End Performance', () => {
```

Comment:
Adding separate files per theme makes it simple with the way the performance report is set up.

I considered an alternative: having a single test for all themes and keys such as `timeToFirstByteTT1`, `timeToFirstByteTT3`, etc. That could also work. The performance command could even have some code to extract those keys into separate tables. However, at that point, the performance command is independent from the actual tests and data. I thought the current approach was simpler to understand and navigate.